### ISSUE TITLE
See also

### ISSUE THREAD

**Issue by github33** [MicrosoftDocs/OfficeDocs-SharePoint#227]
The link for "Search health reports" is for on-prem and does not apply to SharePoint Online.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e4931479-b37a-8350-aba7-13f87937ecc6
* Version Independent ID: 52da4c04-0646-1d3a-39cd-ddce72c2f811
* Content: [View search usage reports](https://docs.microsoft.com/en-us/sharepoint/view-search-usage-reports)
* Content Source: [SharePoint/SharePointOnline/view-search-usage-reports.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointOnline/view-search-usage-reports.md)
* Service: **sharepoint-online**
* GitHub Login: @tklarsen
* Microsoft Alias: **tlarsen**

**Comment by AndreaBarr** [comment]
@github33  Thank you for submitting this feedback.

We apricate you letting us know about the health reports search. I am going to make sure the SharePoint team gets this and addresses it appropriately. 

I was able to get this into the backlog for the SharePoint team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
